lib/takeaway.rb

```diff
@@ -0,0 +1,30 @@
+class Takeaway
+  def initialize(menu)
+    @select = {}
+    @menu = menu
+    @estimate = 0
+  end
+
+attr_reader :menu, :estimate
+
+  def show_menu
+    menu.show
+  end
+
+  def select(order, estimate)
+    @estimate = estimate
+    @select = order
+
+  end
+
+  def correct_bill?
+    raise "incorrect bill amount" unless calculate_bill == estimate
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)